src/containers/image-manager/image-manager.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import ImageManager from '../../components/image-manager/image-manager';
+
+import { cumulus, images as redux_images, selectors } from '@openchemistry/redux';
+
+class ImageManagerContainer extends Component {
+
+  componentDidMount() {
+    this.props.dispatch(redux_images.requestUniqueImages());
+  }
+
+  imageType = () => {
+    // FIXME: this should be set by some internal setting
```

Comment:
Looks like this need to be cleaned up.

Reply:
Well, I was thinking of it as a long-term "FIXME". Marcus said that we will eventually want the type of container to be set inside a config somewhere. But until that happens, we'll just use docker by default.

Reply:
So I think this fix is outside of the scope of this PR.